packages/ui/src/components/ProjectCard/ProjectCard.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Box, SimpleGrid } from '@chakra-ui/react';
+import { Meta, Story } from '@storybook/react';
+
+import { ProjectCard, ProjectCardProps } from './ProjectCard';
+
+const title = 'Zaawansowany React i Node';
+const date = new Date();
+
+const status = { done: 'done', idle: 'idle', doing: 'doing', review: 'review' };
+const meta: Meta = {
+  title: 'Components / ProjectCard',
+  component: ProjectCard,
+  argTypes: {
+    status: {
+      options: Object.keys(status),
+      mapping: status,
+      control: {
+        type: 'select',
+      },
+    },
+  },
+};
+
+export default meta;
+
+const Template: Story<ProjectCardProps> = (args) => (
+  <Box h="444px" w="320px">
+    <ProjectCard {...args} />
+  </Box>
+);
+
+export const Playground = Template.bind({});
+
+Playground.args = {
+  status: 'idle',
+  image: 'test',
```

Comment:
💭 You can provide some real image URL (e.g. from Unsplash) here and in the variants section